clients/logs/eks_s3_logs_client.go

```diff
@@ -149,12 +149,26 @@ func (lc *EKSS3LogsClient) emrLogsToMessageString(run state.Run, lastSeen *strin
 		Key:    aws.String(*key),
 	})
 
-	if s3Obj != nil && err == nil && *s3Obj.ContentLength < int64(10000000) {
+
+	if s3Obj != nil && err == nil {
 		defer s3Obj.Body.Close()
-		gr, err := gzip.NewReader(s3Obj.Body)
-		if err != nil {
-			return "", aws.String(""), errors.Errorf("No driver logs found")
+
+		if s3Obj.ContentLength != nil && *s3Obj.ContentLength < int64(10000000) {
+			return "", aws.String(""), errors.Errorf("Logs > 10MB, will not display.")
```

Comment:
Before users would get `"No driver logs found"` in this case.

We don't have ContentLength for Spark 3.3, so we'll no longer have this check. We could try setting the request headers to avoid the AWS client automatically unzipping, if we really want to keep this check.

Reply:
And I messed up < vs >